Sources/PointFree/Account/Invite.swift

```diff
@@ -181,7 +182,11 @@ private let showInviteLoggedOutView = View<(TeamInvite, User)> { invite, inviter
           ]),
 
         p([`class`([Class.padding([.mobile: [.top: 3]])])], [
-          gitHubLink(text: "Login with GitHub", type: .black, redirectRoute: .invite(.show(invite.id)))
+          gitHubLink(
+            text: "Login with GitHub",
+            type: .black,
+            redirect: url(to: .invite(.show(invite.id)))
```

Comment:
previous this `gitHubLink` helper could take the redirect route, but now it needs the redirect url since it's been put in the `Styleguide` module and no longer has access to the router. I guess I could have made a `gitHubLink` overload in `PointFree` that takes a route...